alvr/client_openxr/src/graphics.rs

```diff
@@ -63,18 +64,33 @@ pub struct ProjectionLayerBuilder<'a> {
     reference_space: &'a xr::Space,
     layers: [xr::CompositionLayerProjectionView<'a, xr::OpenGlEs>; 2],
     alpha: Option<ProjectionLayerAlphaConfig>,
+    composition_layer_settings: Option<xr::sys::CompositionLayerSettingsFB>,
 }
 
 impl<'a> ProjectionLayerBuilder<'a> {
     pub fn new(
         reference_space: &'a xr::Space,
         layers: [xr::CompositionLayerProjectionView<'a, xr::OpenGlEs>; 2],
         alpha: Option<ProjectionLayerAlphaConfig>,
+        clientside_post_processing_config: Option<ClientsidePostProcessingConfig>,
     ) -> Self {
+        let composition_layer_settings = clientside_post_processing_config
+            .map(|post_processing| {
+                xr::CompositionLayerSettingsFlagsFB::from_raw(
+                    (post_processing.sharpening as u64) | (post_processing.super_sampling as u64),
```

Comment:
Half-nit, it's not intuitive how this works if you don't know that `post_processing.sharpening` and `post_processing.super_sampling` are actually compatible bitfields. it's *probably* inferred from the context (we are filling in flags)